runtime/lua/vim/diagnostic.lua

```diff
@@ -657,6 +657,28 @@ local function save_extmarks(namespace, bufnr)
     api.nvim_buf_get_extmarks(bufnr, namespace, 0, -1, { details = true })
 end
 
+---@param opts vim.diagnostic.OptsResolved
+---@return fun(severity: vim.diagnostic.Severity): integer
+local function create_extmark_priority_getter(opts, priority)
```

Comment:
> Also a small nit, but flip priority and opts so that opts is the last parameter (this is more consistent with other code).

:+1: 

> Missing docstring @param for priority.

Damn, there should be a lint rule for that :sweat_smile: